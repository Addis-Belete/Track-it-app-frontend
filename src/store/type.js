export const FETCH_PROGRESS_REQUEST = 'FETCH_PROGRESS_REQUEST';
export const FETCH_PROGRESS_SUCCESS = 'FETCH_PROGRESS_SUCCESS';
export const FETCH_PROGRESS_FAILURE = 'FETCH_PROGRESS_FAILURE';
