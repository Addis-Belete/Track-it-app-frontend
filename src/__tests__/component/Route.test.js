import React from 'react';
import { cleanup, render } from '@testing-library/react';

import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import Routes from '../../component/Route';
import store from '../../store/store';
import '@testing-library/jest-dom';

afterEach(cleanup);
describe('addMeasurment', () => {
  it('should render form to add measurment', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Routes />
        </BrowserRouter>

      </Provider>,

    );
    expect(
      asFragment(
        <Provider store={store}>
          <Routes />
        </Provider>,

      ),
    ).toMatchSnapshot();
  });
});
