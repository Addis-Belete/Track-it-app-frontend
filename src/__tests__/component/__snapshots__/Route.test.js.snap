// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addMeasurment should render form to add measurment 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    Track-app
  </div>
  <div
    class="login"
  >
    <input
      class="input"
      id="name"
      placeholder="Enter Your Name "
      type="text"
      value=""
    />
    <button
      class="button"
      disabled=""
      type="button"
    >
      Login
    </button>
  </div>
</DocumentFragment>
`;
